packages/@lwc/engine/src/framework/api.ts

```diff
@@ -30,9 +30,8 @@ import {
     resolveCircularModuleDependency,
     isCircularModuleDependency,
     EmptyObject,
-    useSyntheticShadow,
 } from './utils';
-import { VM, SlotSet } from './vm';
```

Comment:
`SlotSet` represents the internally allocated vnode tree for each of slot names. This is not longer needed in engine.